html/pfappserver/lib/pfappserver/Form/Config/Network.pm

```diff
@@ -52,12 +52,20 @@ has_field 'dhcp_max_lease_time' =>
   );
 has_field 'ip_reserved' =>
   (
-   type => 'Text',
+   type => 'TextArea',
    label => 'IP Addresses reserved',
    messages => { required => "Range or IP addresses to exclude from the DHCP pool." },
    tags => { after_element => \&help,
              help => 'Range like 192.168.0.1-192.168.0.20 and or IP like 192.168.0.22,192.168.0.24 will be excluded from the DHCP pool' },
   );
+has_field 'ip_assigned' =>
```

Comment:
I suggest putting validation on this field so that if someone mistakenly writes "mange ma bite" in that field (happens if you paste in the PF admin instead of OTBM), then at least it won't be breaking.